docker-compose.f8ui.yml

```diff
@@ -5,6 +5,11 @@ services:
     container_name: fabric8-planner-platform
     image: fabric8-planner-platform
     # command: npm run test:unit
+    environment:
+      # $PROXY_PASS_URL is required by the nginx-config file
+      # It's set by pipeline-library in CI & is emulated here
+      # More info: github.com/fabric8-ui/fabric8-planner/issues/2360
+      PROXY_PASS_URL: "https://api.free-int.openshift.com:443"
```

Comment:
@debloper @jarifibrahim just wanted to know how this would behave on Production?

Reply:
@nimishamukherjee the `docker-compose.f8ui.yml` is for running the OSIO stack locally, for testing purpose and won't be used anywhere in production.
In the pipeline environment, the Jenkinsfile[0] uses the `fabric8-pipeline-library` to set the `PROXY_PASS_URL`[1]. Since we don't have `fabric8-pipeline-library` when running `fabric8-ui` image locally, we need to set this env variable explicitly.

[0] - https://github.com/fabric8-ui/fabric8-planner/blob/master/Jenkinsfile#L1
[1] - https://github.com/fabric8io/fabric8-pipeline-library/blob/master/vars/buildSnapshotFabric8UI.groovy#L37

Reply:
Thanks @jarifibrahim